PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/AdUnit.java

```diff
@@ -494,5 +519,45 @@ public AdUnitConfiguration getConfiguration() {
         return configuration;
     }
 
+    private void registerVisibilityTrackerIfNeeded(BidResponse response) {
+        visibilityMonitor.stopTracking();
```

Comment:
Is it safe that we stop tracking when we receive the bid? The SDK can get a new bid before the impression of the previous one?